ninja-core/src/main/java/ninja/scheduler/Scheduler.java

```diff
@@ -123,8 +126,12 @@ private void schedule(final Object target, final Method method, Schedule schedul
             initialDelay = delay;
         }
 
-        log.info("Scheduling method " + method.getName() + " on " + target + " to be run every " + delay
+        final String targetName = AopUtils.isAopProxy(target)
```

Comment:
Nice! But is there a way to test this functionality? Otherwise someone might remove that functionality and nobody notices until we make a release...

Reply:
Done